float32 left_upper_x
float32 left_upper_y
float32 right_lower_x
float32 right_lower_y
string class

